src/App.tsx

```diff
@@ -1,26 +1,210 @@
 /* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import {
+  USER_ID,
+  addTodo,
+  getTodos,
+  deleteTodo,
+  updateTodo,
+} from './api/todos';
+import { Todo } from './types/Todo';
+import Header from './Cpmponents/Header';
+import TodoList from './Cpmponents/TodoList';
+import Footer from './Cpmponents/Footer';
+import { Status } from './types/status';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodos, setTempTodos] = useState<Todo[]>([]);
```

Comment:
I guess there is only one possible tempTodo at the time, so can be `<Todo | null>(null)`
But not critical 🙂